### PR TITLE
Update Pedal_Box.ino

### PR DESCRIPTION
**Pull request by jackHeun**
Checks for open and short circuit for potentiometer.

### CODE REVIEW

File: ArduinoCode/Pedal_Box/Pedal_Box.ino [hunk 1]
```diff
@@ -134,6 +134,19 @@ void readPotentiometers() {
   accelPin1Val = analogRead(ACCEL_PIN1); 
   accelPin2Val = analogRead(ACCEL_PIN2);
 
+  if(accelPin1Val || accelPin2Val == 0){
```
Comment (jamescd18) [inline ArduinoCode/Pedal_Box/Pedal_Box.ino:137]:
```suggestion
  if(accelPin1Val == 0 || accelPin2Val == 0){
```
Pretty sure that the left and right side of the double pipe or operator both need to be booleans.

File: ArduinoCode/Pedal_Box/Pedal_Box.ino [hunk 2]
```diff
@@ -134,6 +134,19 @@ void readPotentiometers() {
...
+    CAN.sendMsgBuf(CAN_BMS_SHUTDOWN, 0, 8, BMS_ERROR);
+    CAN.sendMsgBuf(CAN_MOTOR, 0, 8, MOTOR_OFF);
+    CAN.sendMsgBuf(CAN_ACCEL_ERROR, 0, 4, ACCEL_ERROR);
+  }
+
+  if(accelPin1Val || accelPin2Val == 1023){
```
Comment (jamescd18) [inline ArduinoCode/Pedal_Box/Pedal_Box.ino:143]:
```suggestion
  if(accelPin1Val == 1023 || accelPin2Val == 1023){
```
Same as above.

**Review by jamescd18 — CHANGES_REQUESTED** [review]
Minor fix needed

File: ArduinoCode/Pedal_Box/Pedal_Box.ino [hunk 3]
```diff
@@ -134,6 +134,19 @@ void readPotentiometers() {
...
+    CAN.sendMsgBuf(CAN_BMS_SHUTDOWN, 0, 8, BMS_ERROR);
+    CAN.sendMsgBuf(CAN_MOTOR, 0, 8, MOTOR_OFF);
+    CAN.sendMsgBuf(CAN_ACCEL_ERROR, 0, 4, ACCEL_ERROR);
+  }
+
+
   if (abs(accelPin1Val - accelPin2Val) <= (1024 * 0.1)) {
```
Comment (rydzynskip) [inline ArduinoCode/Pedal_Box/Pedal_Box.ino:150]: Good catch with the 1023 potentiometer value. This line can also be changed

**Review by rydzynskip — CHANGES_REQUESTED** [review]

**Review by rydzynskip — APPROVED** [review]

**Review by jamescd18 — APPROVED** [review]
oops, yeah, looks good. if I hadn't been so slow with providing review, I would suggest reducing code duplication here by including all four checks OR'ed together in one if block, unless my understanding of the code is wrong

**Pull request merged**